setup.py

```diff
@@ -1,5 +1,31 @@
 from setuptools import setup, find_packages
 
+
+def union(*deps):
+    return list(set().union(*[extras[dep] for dep in deps]))
+
+extras = {
+    "envs": [
+        "atari_py~=0.2.0",            # atari environments
+        "box2d-py~=2.3.5",            # box2d environments
+        "pybullet>=3.0.6",            # continuous environments
+        "Pillow",                     # rendering library
+    ],
+    "test": [
```

Comment:
for whatever its worth calling this test not dev is preferable to me, i just didnt change the name since it was already like that and its not a big deal

Reply:
"dev" is a bit misleading because it's not all of the dev dependencies (e.g., "dev" should include the documentation dependendies)

Reply:
Right, that's why I think test is better

Reply:
oh right, lol